.travis.yml

```diff
@@ -1,42 +1,37 @@
 language: python
-
 addons:
   chrome: stable
-
 matrix:
   include:
-    - python: 3.6
-      env: TOXENV=py36
-    - python: 3.7
-      env: TOXENV=py37
-    - python: 3.8
-      env: TOXENV=py38
-    - python: 3.7
-      env: TOXENV=linting
-    - python: 3.7
-      env: TOXENV=formatting
-    - python: 3.7
-      env: TOXENV=docs
-
-
+  - python: 3.6
```

Comment:
I don't see why you would need to change the indentation here? if any, I think it made it worse :P. From what I could find online, others agree.

Reply:
Same reason as above, I ran it through the travis cli, and it auto-formatted it,I have no strong preference